thentos-core/src/Thentos/Prelude.hs

```diff
@@ -0,0 +1,33 @@
+module Thentos.Prelude (module X) where
+
+import Control.Applicative as X
+import Control.Conditional as X hiding ((|>), (<|), (??))
+-- import Control.Exception.Lifted as X hiding (try)
```

Comment:
I'm ok with using `-XNoImplicitPrelude` globally.  If we can agree on the other open questions.